src/components/bookList.js

```diff
@@ -1,9 +1,16 @@
-import { useSelector } from 'react-redux';
+import { useSelector, useDispatch } from 'react-redux';
+import { fetchBooks } from '../redux/books/books';
 import Book from './book';
 import AddBookForm from './addBookForm';
 
 export default function BookList() {
   const books = useSelector((state) => state.booksReducer);
+  setTimeout(() => {
+    if (books.length === 0) {
+      const dispatch = useDispatch();
+      dispatch(fetchBooks());
+    }
+  }, 1000);
```

Comment:
- [ ] This hook fails and doesn't load the book list from the API.
![Screenshot from 2021-12-18 23-50-07](https://user-images.githubusercontent.com/1642014/146655456-d710c331-d3e6-46ac-bb93-48b34da137eb.png)